Feature/small changes

davidVespa opened the pull request.

davidVespa committed:
Robot sets heading to 90 at the start of autonomous

davidVespa committed:
re-mapped buttons

davidVespa committed:
Added smartDashboard commands
Changed HAB Level 1 X Axis to match center point

davidVespa committed:
Made command to driveToNearestLoadingStation based on the side
added getSide() to SetPoseToFieldLandmarkCommand

davidVespa committed:
added commandgroup to score frontcargo based on side

davidVespa committed:
lines moved

JohnGilb commented on Competition/src/main/java/competition/commandgroups/drivecommandgroups/DriveToNearestLoadingStationCommandGroup.java:
```diff
@@ -0,0 +1,34 @@
+            SetPoseToFieldLandmarkCommand setPose, PoseSubsystem pose,
+            ConfigurablePurePursuitCommand goToLoadingStation) {
+
+        if (setPose.getSide() == Side.Right) {
```
This isn't going to work - the left/right properties are set at the moment of creation. We'll need to talk about how to do this.

davidVespa committed:
Merged 'master'.

davidVespa committed:
Made changes to DriveToNearestLoadingStation
Made new method in PoseSubsystem to getNearestLoadingStation.
Made new test to check if that method works.

davidVespa committed:
Merged 'master'.

davidVespa committed:
Made ScoreOnFrontCargo based on relative position to loading stations
Added commands onto smart dashboard

davidVespa committed:
worked on getPathToNearestLoadingStation

davidVespa committed:
Merged 'master'.

# Conflicts:
#	Competition/src/main/java/competition/operator_interface/OperatorCommandMap.java
#	Competition/src/main/java/competition/subsystems/pose/PoseSubsystem.java

davidVespa committed:
Changed how NearestLoadingStation worked
Figuring out how to score front cargo based on side

davidVespa committed:
"finished" GoToNearestLoadingStation and ScoreOnFrontCargoBasedOnSide
added a way to check what side in SetPoseToFieldLandmark
resolved overload error in commandMap
made getPathToNearestLoadingStation -> NearestLandmark

davidVespa committed:
wip

davidVespa committed:
Made ScoreOnFrontBasedCargo check the side dynamically

davidVespa committed:
started a safemode button

davidVespa committed:
added to smart dashboard

davidVespa committed:
changed safemode

aschokking commented:
It looks like the tests are failing on this branch and the there are a few merge conflicts to take a peek at?

davidVespa committed:
fixed build errors

davidVespa committed:
Merged 'master'.

# Conflicts:
#	Competition/src/main/java/competition/operator_interface/OperatorCommandMap.java

JohnGilb committed:
Merge branch 'master' into feature/smallChanges

JohnGilb committed:
Merge branch 'master' into feature/smallChanges

JohnGilb committed:
Merged 'master'.

# Conflicts:
#	Competition/src/main/java/competition/operator_interface/OperatorCommandMap.java
